lib/livebook_web/components/app_components.ex

```diff
@@ -125,7 +125,7 @@ defmodule LivebookWeb.AppComponents do
             for more information.
           </div>
           <p class="mt-1 text-sm">
-            Automatic clustering is available when deploying to Fly.io and Kubernetes.
+            Automatic clustering is available when deploying to AWS ECS, Fly.io, and Kubernetes.
```

Comment:
```suggestion
            Automatic clustering is available when deploying to AWS ECS (including Fargate), Fly.io, and Kubernetes.
```

@josevalim AWS does a really really really poor job of signaling that Fargate and ECS are the same, I've met a ton of people who had no idea.

Reply:
I pushed that app to keep this one concise. Let's see how it goes. Thank you!